plugin/storage/integration/token_propagation_test.go

```diff
@@ -88,23 +88,25 @@ func TestBearTokenPropagation(t *testing.T) {
 
 	// Run elastic search mocked server in background..
 	// is not a full server, just mocked the necessary stuff for this test.
-	srv := &http.Server{Addr: ":9200"}
+	srv := &http.Server{Addr: ":19200"}
```

Comment:
no reason to run test server on the official ES port (the integration script also changed to run query-service against this fake service)